### PR TITLE
Travis ci

### PR DESCRIPTION
**Pull request by Grummfy**
I have added travis conffiguration.

The build failed, but the idea is to see that easier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/Grummfy/PAGI.svg)](https://travis-ci.org/Grummfy/PAGI)
```
Comment (jacobkiers) [inline README.md]: Will this still work when the PR is merged, or will the build then become available at https://travis-ci.org/marcelog/PAGI?

If so, please change this line.

Comment (Grummfy) [inline README.md]: oups, I change it, right now.

**Comment by jacobkiers** [comment]
Thanks. This PR would make me quite happy too, if merged :smile:

**Comment by marcelog** [comment]
Hey guys!

Looks cool :) but we should make sure that the build passes before merging

**Comment by jacobkiers** [comment]
@marcelog The build will never pass unless you perform the actual integration. That should preferably be done at the Travic CI website itself: https://travic-ci.com/profile.

Alternatively, you can use the github repo settings: https://github.com/marcelog/PAGI/settings/hooks/new?service=travis.

Then and only than can we start testing / building the repository. Pity that you first need the `.travis.yml` file in `master` before it starts working well.

In other words, there seems to be a circular dependency here :)

**Comment by Grummfy** [comment]
By the way, I never have a phpunit fully passed locally ... this is why I add travis, to be sure that nothing locally influence it.

**Comment by Grummfy** [comment]
no news?

**Comment by marcelog** [comment]
Closed with 2.0.0
